AUTHORS.rst

```diff
@@ -0,0 +1,80 @@
+Guidelines
+==========
+
+The purpose of this file is to let newcomers know who is who and to whom best address their
+best address questions or ask a review. The format of this file can also be questionned and
+change, do not hesitate to make suggestions!
+
+Ontology experts
+================
+With Ph.D.
+----------
+
+* github handle (+ real name if relevant/wanted)
+
+ * X years of experience with ontology
+ * Started working on this project since 20XX-XX-XX
+ * Specific relevant area of expertise for this project: ...
+
+
+With Master
+-----------
+
+* github handle (+ real name if relevant/wanted)
+
+ * X years of experience with ontology
+ * Started working on this project since 20XX-XX-XX
+ * Specific relevant area of expertise for this project: ...
+
```

Comment:
Here is an example of how we could populate the file with the existing authors

```suggestion
* @akleinau 

 * X years of experience with ontology
 * Started working on this project since 20XX-XX-XX
 * Specific relevant area of expertise for this project: ...
```